src/LeanMapper/Reflection/PropertyFactory.php

```diff
@@ -50,19 +50,19 @@ public static function createFromAnnotation($annotationType, $annotation, Entity
         $matches = [];
         $matched = preg_match(
             '~
-			^(null\|)?
-			((?:\\\\?[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)+)
-			(\[\])?
-			(\|null)?\s+
-			(\$[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)
-			(?:\s+=\s*(?:
-				"((?:\\\\"|[^"])+)" |  # double quoted string
-				\'((?:\\\\\'|[^\'])+)\' |  # single quoted string
-				([^ ]*))  # unquoted value
-			)?
-			(?:\s*\(([^)]+)\))?
-			(?:\s+(.*)\s*)?
-		~xi',
+            ^(null\|)?
+            ((?:\\\\?[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)+)
+            (\[\])?
+            (\|null)?\s+
+            (\$[a-zA-Z_\x7f-\xff][a-zA-Z0-9_\x7f-\xff]*)
+            (?:\s+=\s*(?:
+                "((?:\\\\"|[^"])+)" |  # double quoted string
+                \'((?:\\\\\'|[^\'])+)\' |  # single quoted string
+                ([^ ]*))  # unquoted value
+            )?
+            (?:\s*\(([^)]+)\))?
+            (?:\s+(.*)\s*)?
+        ~xi',
```

Comment:
No problém je, že ve všech ostatních souborech ([třeba zde](https://github.com/Tharos/LeanMapper/blob/develop/src/LeanMapper/Caller.php)) došlo k změně na 4 mezery a tohle je asi jediné místo, kde zůstaly taby... Nevím v jakém commitu k té změně došlo, samotného mě to včera překvapilo.